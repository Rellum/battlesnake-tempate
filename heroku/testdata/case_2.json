{
  "game": {
    "id": "f77d90fc-5644-462b-a962-372df1205984"
  },
  "turn": 5,
  "board": {
    "width": 11,
    "height": 11,
    "snakes": [
      {
        "id": "gs_4R7gbhhj39GHRmpVHTQpS99G",
        "name": "Hadyn",
        "body": [
          {
            "x": 1,
            "y": 10
          },
          {
            "x": 1,
            "y": 9
          },
          {
            "x": 1,
            "y": 8
          }
        ],
        "health": 95
      }
    ],
    "food": [
      {
        "x": 0,
        "y": 4
      },
      {
        "x": 5,
        "y": 5
      },
      {
        "x": 2,
        "y": 3
      }
    ]
  },
  "you": {
    "id": "gs_4R7gbhhj39GHRmpVHTQpS99G",
    "name": "Hadyn",
    "body": [
      {
        "x": 1,
        "y": 10
      },
      {
        "x": 1,
        "y": 9
      },
      {
        "x": 1,
        "y": 8
      }
    ],
    "health": 95
  }
}