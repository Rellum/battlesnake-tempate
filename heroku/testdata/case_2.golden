Food ⚕: [{0 4} {5 5} {2 3}]
Hadyn ■□□: {gs_4R7gbhhj39GHRmpVHTQpS99G Hadyn 95 [{1 10} {1 9} {1 8}] {0 0} 0   }
◦■◦◦◦◦◦◦◦◦◦
◦□◦◦◦◦◦◦◦◦◦
◦□◦◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦⚕◦◦◦◦◦
⚕◦◦◦◦◦◦◦◦◦◦
◦◦⚕◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
◦◦◦◦◦◦◦◦◦◦◦
